launch_pilot.py

```
@@ -206,8 +615,145 @@ def sync_with_1wp(canister_ids):
+    
+    This function invokes the external thermal simulation script (scripts/thermal_simulation.py) to produce a results file at .pilot/thermal_results.json, reads the JSON output, prints a concise summary of daily metrics (average temperature reduction, energy gain, water saved, estimated revenue), and—if the reported average efficiency gain exceeds 5%—attempts to propagate the improvement to the compute subsystem by calling the hhdao_compute canister to update compute statistics.
+    
+    Returns:
+        dict: Parsed thermal simulation results (contents of .pilot/thermal_results.json) when available.
+        None: If the simulation output is not present or an error occurs.
+    """
+    print("\n🌡️ Running thermal optimization simulation...")
+    
+    try:
+        # Run thermal simulation script
+        result = run_cmd(
```

Ruff (0.13.3)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)